week07_seq2seq/practice_tf.ipynb

```diff
@@ -882,22 +882,22 @@
     "input       y_prev\n",
     "```\n",
     "\n",
-    "__Note:__ you can fit all 4 state tensors of both LSTMs into a in a single state - just assume that it contains, for example, [h0, c0, h1, c1] - pack it in encode and update in decode.\n",
+    "__Note:__ you can fit all 4 state tensors of both LSTMs into an in a single state - just assume that it contains, for example, [h0, c0, h1, c1] - pack it in encode and update in decode.\n",
```

Comment:
Seems that there is still a typo here. Should probably be `in a single state`.

Reply:
Fixed in last commit.